contracts/DonationCore.sol

```diff
@@ -8,9 +8,11 @@ contract DonationCore is ERC721BasicToken, DaiDonation {
   // Soft cap of 4,294,967,295 (2^32-1)
   // E.g. Fill every block for ~50 years
   Donation[] public donations;
+  // Convenience array tracking fundraisers
+  uint256[] public fundraisers;
 
   // Tracking variables
-  uint256 public totalDonationsCreated;
+  uint256 public totalFundraisersCreated;
```

Comment:
This seems like more than nomenclature since there are different objects called Donations and Fundraisers. Are we double sure these are tracking correctly?

Reply:
Unit tests are still passing but I am going to follow up with a full breakout